resolwe/flow/tests/test_filtering.py

```diff
@@ -400,6 +403,14 @@ def test_filter_text(self):
         self._check_filter({'text': 'rst'}, [])
         self._check_filter({'text': 'process'}, self.data)
 
+        # Order by newest first.
+        self._check_filter({'text': 'Data 1'}, [self.data[11], self.data[10], self.data[1]], check_ordering=True)
+        # Order by relevance.
+        # Ordering by empty string should override DataViewSet's default ordering (-created)
+        # and return as ordered by Elastic Search.
+        self._check_filter({'text': 'Data 1', 'ordering': ''}, [self.data[1], self.data[11], self.data[10]],
```

Comment:
There seems to be some randomness here. One of previous force-pushes succeeded with 
`[self.data[1], self.data[11], self.data[10]]` but now wants `[self.data[11], self.data[1], self.data[10]]`

https://travis-ci.org/genialis/resolwe/builds/502007674 (click commit to view code that succeeded)